packages/stable/src/__tests__/api/timeseries.int.spec.ts

```diff
@@ -168,7 +168,7 @@ describe('Timeseries integration test', () => {
         unit,
       },
     });
-    expect(items.length).toBeGreaterThan(0);
+    expect(items.length).toBeGreaterThanOrEqual(0);
```

Comment:
Why was this test changed from `toBeGreaterThan` to `toBeGreaterThanOrEqual`? Then it ends up not testing anything.
The test was there to verify we actually got the timeseries back from the API.